shell/navi.plugin.zsh

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env zsh
 
 _call_navi() {
-   local -r buff="$BUFFER"
-   local -r r="$(printf "$(navi --print </dev/tty)")"
-   zle kill-whole-line
-   zle -U "${buff}${r}"
+  local selected
+  if selected=$(printf "$(navi --print </dev/tty)"); then
```

Comment:
shouldn't we wrap `$(...)` with double quotes in order to include all words?

Reply:
You're right, and it's fixed.